plugins/grpc/cpp/v1.66.1/buf.plugin.yaml

```diff
@@ -4,7 +4,7 @@ plugin_version: v1.66.1
 source_url: https://github.com/grpc/grpc
 description: Generates C++ client and server stubs for the gRPC framework.
 deps:
-  - plugin: buf.build/protocolbuffers/cpp:v27.3
+  - plugin: buf.build/protocolbuffers/cpp:v27.4
```

Comment:
gRPC plugins from today aren't released yet - we can start with a new baseline on protoc 27.4 plugins.